### ISSUE TITLE
Broken localization

### ISSUE THREAD

**Issue by IvanDSM** [bobwya/wine-desktop-common#1]
Sorry for taking a long time to open up the issue. The wine-desktop-common has localization problems. Here's an image for demonstration, on the pt-BR locale:

![image](https://user-images.githubusercontent.com/7508166/36950098-c9dc4a22-1fcf-11e8-9463-ddfaf5647b37.png)

Not only do we have misplaced Chinese (?) characters, "Wine" (which appears as "紅酒" or "vinho") and "Windows" (which is appearing as 視窗) are being translated when they shouldn't be, so instead of "Utilitário de instalação do vinho" we should be seeing "Utilitário de instalação do Wine", and instead of "紅酒 Internet Explorer", it should be "Wine Internet Explorer". "Wine" and "Windows" aren't the only words affected by this bug though, as demonstrated in this other image:

![image](https://user-images.githubusercontent.com/7508166/36950145-72fa248a-1fd0-11e8-91ef-d0ada9c0d8da.png)

Here we get both "Wine" and "boot" translated literally as the drink and the footwear. I can't think of a good replacement for "boot" so I think this word should also remain untranslated in the resulting .desktop files.

Moreover, the .desktop files available in the repository show that this issue isn't limited to the pt-BR locale. It seems that all locales are suffering from this issue. The solution is to ignore the words "Wine", "Windows" and "boot" during the translation process, but I'm not sure how that would be implemented.

**Comment by bobwya** [comment]
Yup, it's just auto-translated with a script (included in repository). It's easy enough to fix some of those issues. I'll being pushing a fairly big clean up of this repository soon - and moving the build script over to python 3.
I'm (actually) just working out fixing some of the **svg** icon sizing, and parsing, at this very moment as it happens...
Obviously I'll leave the issue open until you've verified that the changes are "more accurate".

**Comment by bobwya** [comment]
@IvanDSM 

So, I've finally reached a point where I'm happy that not too many things are broken! I even appear to have learnt how to use some basic Python along the way...

If you want to re-test this package - you can sync my ::**bobwya** Overlay which has new QA Desktop files in the **=app-emulation/wine-desktop-common-99999999** package version. Or just play arouind with a Git checkout... As I've not tagged a new version of this package / repository...

So basically I'll await any comments / criticism from you... Hopefully the translations are a little bit less broken this time!

You probably won't notice any difference with **svg** icons - but I've done a lot of "under-the-hood" tidying up and re-alignment/XML fixes!

**Comment by IvanDSM** [comment]
Hey bob! Sorry for the late response, but I just emerged the 9999999 package and unfortunately, results have been mixed. The first bad thing I noticed was that the items are now in a newly-created category called "Vinho", which means that the category name is now being translated as well.

There are duplicate desktop files for notepad and explorer. For explorer, one of them is not glitched out (It says Wine instead of vinho), and the other one is incorrect. Aside from the explorer desktop file, regedit, task manager, Internet Explorer, winecfg and wineboot are now corrected, showing "Wine" instead of "vinho". The glitch where the asian characters appeared instead of Microsoft Windows is now completely fixed in all desktop files.

Now onto the sad parts :( The desktop files for Wine Help, Notepad, msiexec, Control Panel, Uninstaller, OLE/COM Object Viewer, Minesweeper and Wordpad still have "vinho" instead of Wine.

I also have a suggestion on the placement of the word "Wine" for the pt locale, but I'll leave that for when the desktop files are fully fixed since it's of much lower priority. Thank you for your patience and your work on fixing this! :D

**Comment by bobwya** [comment]
I've been "down in the weeds" over these issues for a while! I think it's getter closer to being correct now...
Would you like to re-test and check what problems are still present?
I've updated the **=app-emulation/wine-desktop-common-99999999** package  QA ignore list for the **.desktop** files...
Thanks again for your input :-)

**Comment by bobwya** [comment]
I have found that Plasma 5 auto-generates a competing local **Wine** (**~/.local/share/applications/wine**) category. This auto-generated category blocks **desktop** files, using a **Wine** category in **/usr/share/applications**.

To work around this ... I've split off a seperate **X-Wine** category for the custom Wine **desktop** files in this repository...

It means you'll probably end up with 2 Wine menu entries: **X-Wine** and **Wine**; but I can't see an easy way around this... :-(

**Comment by bobwya** [comment]
OK minor update... No changes to the translations.

I've remove the separate **X-Wine** category - fixing the need for this with the XDG menu integration files.
Added some **WM_CLASS** hints to the **.desktop** launchers.
Please re-test - if you're still using the package!

NB: the **=app-emulation/wine-desktop-common-99999999::bobwya** ebuild has reverted - with the **QA_DESKTOP_FILE** list re-added.

If there's no further response...
I'll close this issue in 7 days - due to lack of activity.
Assuming it is fixed.

**Comment by bobwya** [comment]
Closing due to lack of activity / feedback.
